### DISCUSSION TITLE
Is there a way to use this directly in a csproj?

### DISCUSSION THREAD

**Discussion by braxtontalbert** [srcnalt/OpenAI-Unity#88] [Q&A] [answered]
I am looking to make a mod for Blade and Sorcery. Typically, you just add the references you need and throw the dll into the standard mod folder format. But I am unsure of how to reference this library, I havent been able to find the dll yet. Any info on this would be awesome, thanks!

**Comment by srcnalt** [comment, marked as answer]
Hi @braxtontalbert,

This project is specifically made for unity with public scripts, and has many UnityEngine.dll dependencies.
If the game is made with Unity, you can compile the Runtime folder's content into a dll and then try giving it a try.

**Reply by braxtontalbert** [reply]
Awesome, I will give that a try. Thanks for the suggestion!

**Reply by braxtontalbert** [reply]
Follow up question. It looks like you are using the UnityEngine.Networking dll. Do you know where to access this to reference ina  project outside of the editor?

**Reply by srcnalt** [reply]
You should be able to find the DLL files in the folder `C:\Program Files\Unity\Hub\Editor\<editor_version>\Editor\Data\Managed\UnityEngine`

**Reply by braxtontalbert** [reply]
Managed to get it working by importing Unity Engine through nuget, but I could not find it locally for some reason. Thanks for the response, though!

**Reply by braxtontalbert** [reply]
Would you mind if I push the dll to this repository? For anyone working in the same kind of situation I am?
